src/core/qgslocaldefaultsettings.cpp

```diff
@@ -37,6 +37,7 @@ void QgsLocalDefaultSettings::setBearingFormat( const QgsBearingNumericFormat *f
   s.endGroup();
 }
 
+
```

Comment:
@nyalldawson , what's that for? ;)

Reply:
the ogc test doesn't trigger unless files in src/core or src/server are touched. I'll roll it back if the test is successful.